app.py

```diff
@@ -16,3 +16,10 @@ def get_toid_building(input):
 def get_feature(input):
     return jsonify(features.get_feature_from_toid(input))
 
+@app.route("/features/ngd/<input>")
```

Comment:
Do we need both? If we're deprecating the old one, remove the Ngd part of the route, and method name, and use that one in place